### PR TITLE
[3D] Fix missing Z support for point layers

### PR DESCRIPTION
**Pull request by nirvn**
## Description
A picture worth a thousand words:
screenshot from 2017-11-08 17-26-40

@wonder-sk , your code is so clean, real easy to add to it. Review most appreciated.

FYI, this takes care of a //TODO: , adding Z support for points layers (in addition to Z support already present in line and polygon layers).

## Checklist

> Reviewing is a process done by project maintainers, mostly on a volunteer basis. We try to keep the overhead as small as possible and appreciate if you help us to do so by completing the following items. Feel free to ask in a comment if you have troubles with any of them.

- [x] Commit messages are descriptive and explain the rationale for changes
- [x] Commits which fix bugs include `fixes #11111` in the commit message next to the description
- [x] Commits which add new features are tagged with `[FEATURE]` in the commit message
- [x] Commits which change the UI or existing user workflows are tagged with `[needs-docs]` in the commit message and containt sufficient information in the commit message to be documented
- [x] I have read the [QGIS Coding Standards](https://docs.qgis.org/testing/en/docs/developers_guide/codingstandards.html) and this PR complies with them
- [x] This PR passes all existing unit tests (test results will be reported by travis-ci after opening this PR)
- [x] New unit tests have been added for core changes
- [x] I have run [the `scripts/prepare-commit.sh` script](https://github.com/qgis/QGIS/blob/master/.github/CONTRIBUTE.md#contributing-to-qgis) before each commit

### CODE REVIEW

**Comment by NathanW2** [comment]
Yessssssssss!!!!!!!

File: src/3d/symbols/qgspoint3dsymbol.cpp [hunk 1]
```diff
@@ -20,6 +20,7 @@
 
 
 QgsPoint3DSymbol::QgsPoint3DSymbol()
+  : mAltClamping( AltClampRelative )
```
Comment (wonder-sk) [inline src/3d/symbols/qgspoint3dsymbol.cpp]: Best to use modern way of member initialization in the class definition (see e.g. how mShape is initialized)

**Review by wonder-sk — APPROVED** [review]
Looks great!

File: src/3d/symbols/qgspoint3dsymbol.cpp (hunk 1 above)
Comment (nirvn) [reply src/3d/symbols/qgspoint3dsymbol.cpp]: @wonder-sk , I was copying the style found in qgsline3dsymbol and qgspolygon3dsymbol. Should I stick to that, or update all three to the modern way?

Comment (wonder-sk) [reply src/3d/symbols/qgspoint3dsymbol.cpp]: I'm easy - either way works for me :-)

**Pull request merged**